_pages/software-tools/slack.md

```diff
@@ -310,10 +310,16 @@ If you're interested in learning more about the bots, or perhaps contributing to
 ### Channel naming conventions
 
 - Channels that begin with `admin-` include administrators for various tools. [#admins-slack](https://gsa-tts.slack.com/messages/admins-slack/), for example, is used to request invites to Slack (see above), expunge a particularly offensive/off-topic message (see above), change the name of an existing channel, and so on.
-- Channels that begin with `wg-` are for working groups; channels that begin with `g-` are for guilds. Feel free to join a working group or guild channel even if you are not actively part of the group.
 - Channels that end with `-partner` include partners from other agencies.
 - Channels that end with `-public` may include members of the public.
 
+Channels focused on TTS practices have their own conventions:
```

Comment:
@tbaxter-18f This might be a good time to start [using slack channel prefixes ](https://slack.com/help/articles/217626408-Create-guidelines-for-channel-names#create-channel-prefixes) so we might want to align the change with that as well. 

Also, want might want to get some design input to see if there might be any cognitive issues with the current prefixes; especially because cloud.gov uses 'cg-' so  it could start getting some folks confused with the redundant letters being used

Reply:
all of the conventions other than the new `c-` convention for communities have been in place for years.